tests/test_docker_publish_workflow.py

```
@@ -0,0 +1,40 @@
+            index for index, line in enumerate(self.lines) if qemu_marker in line
+        )
+        buildx_index = next(
+            index for index, line in enumerate(self.lines) if buildx_marker in line
+        )
+
+        self.assertIn(qemu_marker, self.content)
+        self.assertIn(buildx_marker, self.content)
+        self.assertLess(qemu_index, buildx_index)
+
+    def test_workflow_builds_amd64_and_arm64_images(self) -> None:
+        self.assertIn("platforms:", self.content)
```

Ruff (0.15.5)

[warning] Use a regular `assert` instead of unittest-style `assertIn`

Replace `assertIn(...)` with `assert ...`

(PT009)